code/modules/mob/living/carbon/death.dm

```diff
@@ -1,3 +1,36 @@
+#define TOXIN_TO_INTERNAL_DAMAGE_MULTIPLIER 2
+
+// This function applies damage to internal organs in case the mob died having toxin damage
+// This feature increases amount of effort it takes to revive a victim of poisoning
```

Comment:
Используй такой комментарий на проках как минимум. Эти комментарии будет подсвечивать, если установлен dm-langserver и кто-то наведет мышкой на прок, там где он вызывается.
Ну и вообще так принятно комментить. 
```suggestion
/**
* This function applies damage to internal organs in case the mob died having toxin damage
* This feature increases amount of effort it takes to revive a victim of poisoning
*/
```